06-rmarkdown.Rmd

```diff
@@ -8,20 +8,44 @@ ottrpal::set_knitr_image_path()
+The following are reasons why notebooks help reproducibility:
 
-- They allow you to show and share your code and the output of your code in one place! (this can be done in several ways depending on what you want)
+- They allow you to show and share your code and the output of your code in one place! (This can be done in several ways depending on what you want.)
 - They allow you to test if your code works outside of what is active in your environment
-- They allow you to test sections and all previous sections of your code out to troubleshoot 
+- They allow you to test sections and all previous sections of your code, which can help with troubleshooting
 - They help you understand what might be wrong with your code in smaller sections of code if you have an issue
 
-## Getting Started with R Markdown
+## R Markdown or Quarto?
 
-OK, so now we know how to make a new R Markdown file from the previous section.
+Both R Markdown and Quarto are types of notebooks that have similar functions. R Markdown files end with the suffix `.Rmd` while quarto files end with `.qmd`.
+
+Both Qmd and Rmd files are both notebooks that have the benefits we've described above. They allow you to document using the markdown language. Plus, because they are so similar you can often just change the suffix of your file and convert between these file types (results may vary depending on the content of the file).
+
+R Markdown was the first R programming notebook on the scene, and has a lot of tools devoted to it because it has been around awhile. In 2022, [Posit released the Quarto notebook](https://posit.co/blog/announcing-quarto-a-new-scientific-and-technical-publishing-system/). So Quarto has a lot of great new features but is still relatively new.
+
+Posit created Quarto with the idea of streamlining document making and allowing for more compatibility with languages that are not R. While R Markdown documents also (sort of) allow for other languages to be run in them, their ability to do this successfully is limited.
+
+### R Markdown Pros:
+- Time tested, a lot of packages and resources built for it.
+- Fundamentally an R notebook and is built around that.
+
+### R Markdown Cons:
+- Does not always do well running other languages (like Python).
+- Does require a lot of extra packages to be installed to do more things with it: `bookdown`, `distill`, etc.
+
+### Quarto Pros:
+- Built with more compatibility for other languages
+- Appears to be more streamlined/centralized and less need for a lot of extra packages to create other types of documents.
+
+### Quarto Cons:
+- It is still quite new, and the community is still catching up to it. Though it appears to be built with backwards compatibility in mind.
+- Because it so new, there are still features they are continuing to build for it that R Markdown has that it may not just yet. At this point, these are mostly very fringe features, but it could be something an individual user is interested in.
```

Comment:
```suggestion
- Because it so new, there are still some features that are being developed for Quarto that R Markdown already supports. At this point, these are mostly features that would allow for customization.
```